### DISCUSSION TITLE
Update: Tue, Sep 28, 2021

### DISCUSSION THREAD

**Discussion by hansmrtn** [moonshotcollective/quadratic-diplomacy#10] [General]
Today the QDip team had a discussion regarding the Friday Memepolooza event, and the new mono-repo workflow. 

Here is an outline of some topics we discussed: 

- For this Friday, we will use [off-chain quadratic diplomacy](https://github.com/carletex/quadratic-diplomacy/tree/votes-off-chain) to demo with Memepolooza 🥳  - It is gasless, fast, and effective! We should deploy this to mainnet before Friday. 
- We will use the monorepo to integrate the event-driven and on-chain versions into an integrated UI and contract. The user will be able to select between two main versions, "Qdip-lite" (event-driven, gasless votes) and "Qdip-heavy" (on-chain). Focus on these two, and add new versions in the future (metatransactions, ceramic/ipfs, etc.) for the user to choose, if necessary.

**Comment by over9001** [comment]
I like this direction and sharpening of focus! Keep up the good work! ^_^

**Comment by damianmarti** [comment]
The contract is the same from the on-chain events votes, so if this contract is already deployed to mainnet, we can use it.
